en/docs/install-and-setup/setup/sso/configuring-external-idp-using-oidc-for-multi-tenancy.md

```
@@ -30,36 +34,38 @@ WSO2 API Manager uses OIDC SSO by default. This guide explains how to connect WS
         ```
 
-## Configure Identity Server
+## Configure the Identity Server
 
-### Step 1 - Create New Organization
+### Step 1: Create new Organization
 
-1.  Sign in to WSO2 IS Management Console:
+1.  Sign in to the Management Console of WSO2 IS by browsing the following URL:  
 
     ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)